introduction.md

```diff
@@ -0,0 +1,4 @@
+Hi, my name is Sergio
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
ok, i added it to introduction.md